src/clr-angular/utils/popover/adapter/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export * from './popover-adapter';
```

Comment:
I don't think this should be part of the public API. It is an internal only component. 

Having it exported from the index means it will get picked up when the golden file is built.

Reply:
There was no need for the file after all